driver-core/src/test/java/com/datastax/driver/core/CCMBridge.java

```diff
@@ -40,7 +40,11 @@
 
     private static final Logger logger = LoggerFactory.getLogger(CCMBridge.class);
 
-    private static final String CASSANDRA_VERSION;
+    private static final String INPUT_CASSANDRA_VERSION;
```

Comment:
This is actually only used in the static block below and could be turned into a local variable.